draft-ietf-quic-tls.md

```diff
@@ -826,8 +826,8 @@ in Section 5.3 of {{!TLS13}}, the IV length is the larger of 8 or N_MIN (see
 Section 4 of {{!AEAD=RFC5116}}; all ciphersuites defined in {{!TLS13}} have
 N_MIN set to 12).
 
-The size of the packet protection key is determined by the packet protection
-algorithm, see {{pn-encrypt}}.
+The size of the packet number protection key is determined by the packet number
+protection algorithm, see {{pn-encrypt}}.
```

Comment:
The existing statement is true as well.  Perhaps say that the size of both keys depends on the negotiated AEAD?

Reply:
So, the way I read it, is that key *and IV* length (i.e. key_length and iv_length) is actually defined in the paragraph before, but nowhere it says how long is the packet number encryption key (i.e. pn_key_length) which is what confused me (also considering the "see {{pn-encrypt}}" which didn't make a whole lot of sense to me since packet number encryption wasn't mentioned at all in that sentence).

Reply:
Though I'm fine adding both, it just seems superfluous.